src/Http/ServerRequestFactory.php

```diff
@@ -2,22 +2,33 @@
 namespace CakeDC\Roadrunner\Http;
 
 use Cake\Core\Configure;
-use Cake\Error\Debugger;
 use Cake\Http\ServerRequest;
+use Cake\Http\ServerRequestFactory as BaseServerRequestFactory;
+use function Laminas\Diactoros\normalizeServer;
 
-class ServerRequestFactory extends \Cake\Http\ServerRequestFactory
+/**
+ * Creates an instance of ServerRequest.
+ */
+class ServerRequestFactory extends BaseServerRequestFactory
```

Comment:
This class is currently only used in tests, `Bridge` directly uses `Cake\Http\ServerRequestFactory`. So is it really required?

Reply:
I don't think so, it does make use of `CakeDC\Roadrunner\Http\Session` though. I haven't gotten to testing sessions and an actual real cakephp app with this though, just testing on cakephp welcome page so far.